libraries/kmp-iap/library/src/androidMain/kotlin/io/github/hyochan/kmpiap/InAppPurchaseAndroid.kt

```
@@ -1760,6 +1841,303 @@ internal class InAppPurchaseAndroid : KmpInAppPurchase, Application.ActivityLife
+                                PurchaseError(
+                                    code = ErrorCode.Unknown,
+                                    message = "Billing Choice info failed: ${result?.debugMessage}"
+                                )
+                            ))
+                        }
+                    }
+                    null
+                }
+                client.javaClass.getMethod("getBillingChoiceInfoAsync", paramsClass, listenerClass)
+                    .invoke(client, requestParams, listener)
+            } catch (e: NoSuchMethodException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)